app/code/Magento/InventoryImportExport/Model/Import/Serializer/LegacyJsonHelperInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventoryImportExport\Model\Import\Serializer;
+
+/**
+ * Extension point for row validation
+ *
+ * @api
+ */
+interface LegacyJsonHelperInterface
```

Comment:
why do we need to introduce this interface? especially if it's already legacy? 

If you want to make Backward Compatible change, legacy class 
`Magento\Framework\Json\Helper\Data` should be adapted as well.

Because currently, you are breaking the contract, as AbstractEntity accept 
 `      \Magento\Framework\Json\Helper\Data $jsonHelper,`
But you pass 
` SerializerInterface $jsonHelper, ` instead